src/components/Home.jsx

```diff
@@ -3,23 +3,37 @@ import {
   Typography, Row, Col, Statistic,
 } from 'antd';
 import millify from 'millify';
+import { Link } from 'react-router-dom';
 import { useGetCryptosQuery } from '../services/cryptoApi';
+import About from './About';
+import CoinDetails from './CoinDetails';
 
 const { Title } = Typography;
 
 const Home = () => {
-  const { data } = useGetCryptosQuery();
+  const { data, isFetching } = useGetCryptosQuery();
   const globalStats = data?.data?.stats;
-  console.log(data);
+  if (isFetching) return <div>Loading...</div>;
   return (
     <>
       <Title level={2} className="heading"> Ger Crypto Stats</Title>
       <Row>
         <Col span={12}><Statistic title="All Crypto Currencies" value={globalStats.total} /></Col>
-        <Col span={12}><Statistic title="All Crypto Currencies" value={millify(globalStats.totalExchanges)} /></Col>
-        <Col span={12}><Statistic title="All Crypto Currencies" value="5" /></Col>
-        <Col span={12}><Statistic title="All Crypto Currencies" value="5" /></Col>
+        <Col span={12}><Statistic title="Total Exchanges" value={globalStats.totalExchanges} /></Col>
+        <Col span={12}><Statistic title="Total Market Cap" value={millify(globalStats.totalMarketCap)} /></Col>
+        <Col span={12}><Statistic title="Total Volume" value={millify(globalStats.total24hVolume)} /></Col>
+        <Col span={12}><Statistic title="Total Markets" value={millify(globalStats.totalMarkets)} /></Col>
```

Comment:
- [ ] Good job so far 👍 Kindly apply the same styling and colors as found in the original design. Pay special attention to the fact that in the mobile view, the items should be 2 in a row and the colors should be alternating 💯 
- Your design:
![Annotation 2022-06-24 164301](https://user-images.githubusercontent.com/85113567/175573809-9570be10-b6da-4de9-bbea-db09cc020287.jpg)
- Required design:
![Annotation 2022-06-24 164326](https://user-images.githubusercontent.com/85113567/175573837-d64529e5-7089-456e-b84f-724e057fbccc.jpg)

Kindly do this for both the home and the details pages 👍